File: app/src/main/java/io/github/romangolovanov/apps/ametro/utils/ModelUtils.java
package io.github.romangolovanov.apps.ametro.utils;

import android.graphics.Rect;

import io.github.romangolovanov.apps.ametro.model.entities.MapRect;

public class ModelUtils {

    public static Rect toRect(MapRect r) {
        if (r == null) return null;
        return new Rect(r.x, r.y, r.x + r.width, r.y + r.height);
    }

}

File: app/src/main/java/io/github/romangolovanov/apps/ametro/utils/ModelUtils.kt
package io.github.romangolovanov.apps.ametro.utils

import android.graphics.Rect
import io.github.romangolovanov.apps.ametro.model.entities.MapRect

object ModelUtils {

    @JvmStatic
    fun toRect(r: MapRect?): Rect? {
        if (r == null) return null
        return Rect(r.x, r.y, r.x + r.width, r.y + r.height)
    }
}
